ArduCopter/motor_test.cpp

```diff
@@ -65,13 +65,13 @@ void Copter::motor_test_output()
                 if (motor_test_throttle_value <= 100) {
                     int16_t pwm_min = motors->get_pwm_output_min();
                     int16_t pwm_max = motors->get_pwm_output_max();
-                    pwm = pwm_min + (pwm_max - pwm_min) * (float)motor_test_throttle_value/100.0f;
+                    pwm = (int16_t) (pwm_min + (pwm_max - pwm_min) * motor_test_throttle_value * 1e-2f);
```

Comment:
ideally, pwm should be uint16_t and AP_motors output_test_seq() should input uint16_t too as that what will be use on SRV library to write the final output

Reply:
I didn't chase that, as it was already calculating it as a `int16_t` :) (and the actual range we are using is well inside of that)

Reply:
I don't think that is blocking but that would be a nice change for static analyser as they will complain numerous time that we go from signed to unsigned whereas in Copter/motor_test.cpp they will cry only once ^^